unlock-app/src/hooks/usePostmateParent.ts

```diff
@@ -1,11 +1,11 @@
 import { useState, useEffect } from 'react'
-import { ChildAPI, Model } from 'postmate'
+import Postmate from 'postmate'
```

Comment:
It didn't like the destructuring import when actually running, so I changed it back to this